api/src/modules/business-units/business-unit.repository.ts

```diff
@@ -69,13 +72,22 @@ export class BusinessUnitRepository extends ExtendedTreeRepository<
         'sl.scenarioInterventionId = scenarioIntervention.id',
       );
 
-      queryBuilder.andWhere(
-        new Brackets((qb: WhereExpressionBuilder) => {
-          qb.where('scenarioIntervention.scenarioId IN (:...scenarioIds)', {
-            scenarioIds: businessUnitTreeOptions.scenarioIds,
-          }).orWhere('sl.scenarioInterventionId is null');
-        }),
-      );
+      queryBuilder
+        .andWhere(
+          new Brackets((qb: WhereExpressionBuilder) => {
+            qb.where('sl.scenarioInterventionId is null').orWhere(
+              new Brackets((qbInterv: WhereExpressionBuilder) => {
+                qbInterv
+                  .where('scenarioIntervention.scenarioId IN (:...scenarioIds)')
+                  .andWhere(`scenarioIntervention.status = :status`);
+              }),
+            );
+          }),
+        )
+        .setParameters({
```

Comment:
Any specific reason to use another approach to set parameters other than we are used so far?
i.e 
```
.where('scenarioIntervention.scenarioId IN (:...scenarioIds)', { scenarioIds: dto.scenarioIds })
```

If there is no other reason I believe we should be consistent